ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -10,78 +17,91 @@
     - name: Filter out edpm vm
       ansible.builtin.set_fact:
         ssh_key_file: "{{ cifmw_artifacts_basedir }}/artifacts/edpm/ansibleee-ssh-key-id_rsa"
+        ssh_user: root
         edpm_vms: >-
           {%- set listing=vms_list.list_vms | select('match', '^edpm-.*$') -%}
           {%- if listing | length == 1 -%}
           ['192.168.122.100']
           {%- endif -%}
-  rescue:
-    - name: Check for network info env file
-      register: network_env_file
+
+- name: Hit compute nodeset
+  when:
+    - "'libvirtd' not in ansible_facts.services"
+  block:
+    - name: Check for CI env directory
+      register: network_env_dir
       ansible.builtin.stat:
-        path: /etc/ci/env/networking-info.yml
+        path: /etc/ci/env
 
     - name: Extract data from network env file if available
       when:
-        - network_env_file.stat.exists
+        - network_env_dir.stat.exists
       block:
         - name: Load network env file
           ansible.builtin.include_vars:
-            file: /etc/ci/env/networking-info.yml
+            dir: /etc/ci/env/
 
         - name: Extract Compute from zuul mapping if any
           when:
             - crc_ci_bootstrap_networks_out is defined
-            - cifmw_edpm_deploy_extra_vars
+            - cifmw_edpm_deploy_extra_vars is defined
           ansible.builtin.set_fact:
             ssh_key_file: "{{ cifmw_edpm_deploy_extra_vars.SSH_KEY_FILE }}"
+            ssh_user: zuul
```

Comment:
This one is safe, but we are using this value before hardcoding it to zuul: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/hooks/playbooks/fetch_compute_facts.yml#L168

Reply:
wondering about multiple compute support though... Maybe `{{ ansible_user }}` would be enough actually?